types/index.d.ts

```diff
@@ -1053,3 +1053,40 @@ export function decodeAddressToMuxedAccount(address: string, supportMuxing: bool
 export function encodeMuxedAccountToAddress(account: xdr.MuxedAccount, supportMuxing: boolean): string;
 export function encodeMuxedAccount(gAddress: string, id: string): xdr.MuxedAccount;
 export function extractBaseAddress(address: string): string;
+
+export type ScIntType =
+  | 'i64'
+  | 'u64'
+  | 'i128'
+  | 'u128'
+  | 'i256'
+  | 'u256';
+
+export class XdrInt {
+  constructor(
+    type: ScIntType,
+    values: number|bigint|string|ScInt|Array<number|bigint|string|ScInt>
```

Comment:
Oops, you can't pass a `ScInt` to an `XdrInt` :facepalm: 

```suggestion
    values: number|bigint|string|Array<number|bigint|string>
```